consensus/pbft/messages.proto

```diff
@@ -36,15 +36,15 @@ package pbft;
 
 message message {
     oneof payload {
-        request request = 1;
+        request_batch request_batch = 1;
         pre_prepare pre_prepare = 2;
         prepare prepare = 3;
         commit commit = 4;
         checkpoint checkpoint = 5;
         view_change view_change = 6;
         new_view new_view = 7;
-        fetch_request fetch_request = 8;
-        request return_request = 9;
+        fetch_request_batch fetch_request_batch = 8;
+        request_batch return_request_batch = 9;
```

Comment:
Agreed this is the least invasive way for now, but something to consider for future work.